content/configuration/resolve.md

```diff
@@ -162,3 +234,22 @@ W> Changes to cached paths may cause failure in rare cases.
 `object`
 
 This set of options is identical to the `resolve` set above, but is used only to resolve webpack's [loader](/concepts/loaders) packages.
+
+Default:
```

Comment:
Always use a newline before a codeblock. Some markdown parsers go nuts if you don't do this.